### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by PTI-Kerrie**
These are just all the little things Grammarly was able to detect.

### CODE REVIEW

File: imaging.rst [hunk 1]
```diff
@@ -183,4 +183,4 @@ QR Code® is a registered trademark of DENSO WAVE INCORPORATED.
     :Example Print logo 2 from dotlines 0 to 861:
         .. code-block:: none
 
-            write('\x1b\xfa\x02\x00\x00\x03\x5e')   #  862 dotlines total
```
Comment (ghost) [inline imaging.rst:186]: I don't see a diff here, might a whitespace thing. Did you happen to render this and verify that it looks okay?

File: paper_movement.rst [hunk 2]
```diff
@@ -91,7 +91,7 @@ presenting and retracting.
         +-----+--------------------------+-----------------------------------------------------------------------------------------------------------------+
         | 20  | None                     | Enable dispenser continuous mode                                                                                |
         |     |                          |                                                                                                                 |        
-        |     |                          | - While printing, the ticket is continuously pushed from outlet                                                 |
+        |     |                          | - While printing, the ticket is continuously pushed from the outlet                                                 |
```
Comment (ghost) [inline paper_movement.rst]: Does this extra word extend the table width?

**Review by ghost — APPROVED** [review]
Thanks for this!

File: imaging.rst (hunk 1 above)
Comment (PTI-Kerrie) [reply imaging.rst:186]: It views ok,  I couldn't figure out how to reject the changes completely.

File: paper_movement.rst (hunk 2 above)
Comment (PTI-Kerrie) [reply paper_movement.rst]: This does extend the table, I'll try and clear that up

Comment (ghost) [reply paper_movement.rst]: Okay thank you. RST tables are just super sensitive to spacing unfortunately.

File: imaging.rst (hunk 1 above)
Comment (ghost) [reply imaging.rst:186]: Awesome, don't worry about this one then.

**Pull request merged**